packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -118,7 +118,19 @@ export class EdgelessPageBlockComponent
   }
 
   private _removeHotkeys() {
-    hotkey.removeListener([HOTKEYS.BACKSPACE], this.flavour);
+    hotkey.removeListener([
```

Comment:
These listeners seem should be removed. I'm not quite sure if it's careless or intentional here.